.gitignore

```diff
@@ -0,0 +1,105 @@
+
+# Created by https://www.gitignore.io/api/python
```

Comment:
Make sure you also include your OS and editor in the `.gitignore` to avoid getting `.DS_Store` and `.vscode` files.